taskana-adapter-camunda-system-connector/src/main/java/pro/taskana/adapter/systemconnector/camunda/config/CamundaSystemUrls.java

```diff
@@ -9,39 +9,46 @@
 @Component
 public class CamundaSystemUrls {
 
-  Set<SystemUrlInfo> theCamundaSystemUrls = new HashSet<>();
+  Set<SystemUrlInfo> camundaSystemUrls = new HashSet<>();
 
   public CamundaSystemUrls(String strUrls) {
```

Comment:
should we add a dedicated unit test for the initialisation?

Reply:
good idea, I'm gonna open a ticket for that

Reply:
ok